docker/third-party/images.txt

```diff
@@ -14,6 +14,6 @@ python:3.10-slim
 redis:6.0.6-alpine
 ubuntu:18.04
 ubuntu:19.04
-ubuntu:20.04
+ubuntu:22.04
 ubuntu:bionic-20200921
 ubuntu:focal-20221019
```

Comment:
Remove focal here as well. I think we can nix all the old ubuntu's. @daniel-goldstein @jigold objections to that?

Reply:
I think we should leave `20.04`. The others are fine to nix.

Reply:
bump: do we need to keep 20.04? Presumably we can kill 18.04 and 19.04.

Reply:
See above message, we should keep `ubuntu:20.04` until we migrate the `global-config` then we can delete it.

Reply:
And we'll delete 19.04 then too, right?